documentation/staging/content/userguide/base-images/access-images.md

```diff
@@ -0,0 +1,115 @@
+
+- The name of the secret must be added to these domain resources using
+  the `domain.spec.imagePullSecrets` field. For example:
+
+  ```text
+  ...
+  spec:
+  ...
+    imagePullSecrets:
+    - name: SECRET_NAME
+  ...
+  ```
+
+- If you are using the Oracle Container Registry, then
+  you must use the web interface to accept the Oracle Standard Terms and Restrictions
+  for the Oracle software images that you intend to deploy.
+  You need to do this only once for a particular image.
+  See [Obtain images from the Oracle Container Registry](#obtain-images-from-the-oracle-container-registry).
+
+For more information about creating Kubernetes Secrets for accessing
+the registry, see the Kubernetes documentation,
+[Pull an image from a private registry](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/).
+
+#### Option 2: Store images in a central registry and set up a Kubernetes service account with image pull secrets in each domain namespace
+
+An option for accessing an image that is stored in a private registry
+is to set up the Kubernetes `ServiceAccount` in the namespace running the
+WebLogic domain with a set of image pull secrets thus avoiding the need to
+set `imagePullSecrets` for each `Domain` resource being created (because each resource
+instance represents a WebLogic domain that the operator is managing):
+
+- Create a Kubernetes `docker-registry` secret as shown
+  in [Option 1](#option-1-store-images-in-a-central-registry-and-set-up-image-pull-secrets-on-each-domain-resource).
+
+- Modify the `ServiceAccount` that is in the same namespace
+  as your domain resources to include this image pull secret:
+
+  ```shell
+  $ kubectl patch serviceaccount default -n domain1-ns \
+  -p '{"imagePullSecrets": [{"name": "my-registry-pull-secret"}]}'
+  ```
+
+  Note that this patch command entirely replaces the current list of
+  image pull secrets (if any). To include multiple secrets, use
+  the following format:
+  `-p '{"imagePullSecrets": [{"name": "my-registry-pull-secret"}, {"name": "my-registry-pull-secret2"}]}'`.
+
+For more information about updating a Kubernetes `ServiceAccount`
+for accessing the registry, see the Kubernetes documentation,
+[Configure service accounts for pods](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-image-pull-secrets-to-a-service-account).
+
+#### Option 3: Manually place images on Kubernetes cluster nodes
+
+Alternatively, it may be preferable to manually place an image
+on each Kubernetes worker node in your Kubernetes cluster in advance.
+
+For example, if the desired image is located in a Docker registry,
+then you can manually call `docker login` and `docker pull` on each
+worker node. For the steps with Oracle Container Registry images,
+see [Obtain images from the Oracle Container Registry](#obtain-images-from-the-oracle-container-registry).
+
+As another example,
+if the Docker image is located in a local Docker cache,
```

Comment:
We've been slowly moving away from the term "Docker" except when referring specifically to that product. I think that "Docker image" is a place where we can globally move to "container image" or just "image" regardless if the context is about the Docker product. The same thing applies to "Docker registry", which can globally be "container registry".

Reply:
In my opinion, `image registry` is a more accurate term...

Reply:
I agree with both suggestions and will edit the doc accordingly.

Reply:
Yeah, I agree about "image registry" except that my Google search suggests that most others have chosen "container registry", e.g. Azure Container Registry, Oracle Container Registry.

Reply:
@rjeberhard Need clarification here: do you want me to search and replace all instances in the documentation of Docker image with "container image" or just "image"?

Reply:
It looks like you are already using "image" in most places, so I think that it is unnecessary that these become "container image". The full "container image" only seems necessary if there is a possible ambiguity with the use of just "image". That is likely not the case in this document, so I'd leave them all as "image". We could generally have a first use of "container image" followed after by uses of "image", but I don't think that's necessary in this document as the context is clear.

Reply:
yeah, ok...

Reply:
@rjeberhard Yes, for this document. However, we use the term, Docker registry, throughout the doc set. Do you want me to change them accordingly?

Reply:
Yes, please.

Reply:
@rjeberhard FYI. Done, except for this AKS user guide document, https://oracle.github.io/weblogic-kubernetes-operator/userguide/aks/#configure-aks-cluster, which, as seen in the second table, **Field** column, has this field name, "Use a pre-existing WebLogic Server Docker image from Oracle Container Registry (OCR)?" I did not change anything that was related to this field.